src/Swarm/Web.hs

```diff
@@ -25,41 +25,51 @@ import Control.Exception (Exception (displayException), IOException, catch, thro
 import Control.Lens ((^.))
 import Control.Monad (void)
 import Control.Monad.IO.Class (liftIO)
+import Data.Foldable (toList)
 import Data.IORef (IORef, readIORef)
 import Data.IntMap qualified as IM
 import Data.Maybe (fromMaybe)
+import Data.Text qualified as T
 import Network.Wai qualified
 import Network.Wai.Handler.Warp qualified as Warp
 import Servant
 import Swarm.Game.Robot
 import Swarm.Game.State
+import Swarm.TUI.Model
 import System.Timeout (timeout)
 
 type SwarmApi =
   "robots" :> Get '[JSON] [Robot]
     :<|> "robot" :> Capture "id" Int :> Get '[JSON] (Maybe Robot)
+    :<|> "repl" :> Get '[JSON] [T.Text]
```

Comment:
Maybe we should add a length (or start+end) parameter?

My history is quite long, so I imagine it might be useful to shorten it if the Swarm binary was not running on my machine.

Or alternatively, how about only showing the history from the current game (as we do in REPL) unless you explicitly request it?

Reply:
I've now made the path longer for the current output (`repl/history/full`) to future-proof for different varations/parameters.